src/providers/definition/common.ts

```diff
@@ -119,7 +119,7 @@ export const viewSuggestions: DefinitionSuggestion[] = [
 			return new RegExp(`name=["']${text}["']>(.*)?</`, 'g');
```

Comment:
<!--
  0 failure: 
  1 warning:  src/providers/def...
  
  
  DangerID: danger-id-Danger;
  File: src/providers/definition/common.ts;
  Line: 119;
-->

- :warning: src/providers/definition/common.ts line 119 – Found non-literal argument to RegExp Constructor (security/detect-non-literal-regexp)